### DISCUSSION TITLE
extention method

### DISCUSSION THREAD

**Discussion by itafra-net** [IT-Afra/bakery-shop#63] [Q&A] [answered]
According to the principles of onion architecture, it is necessary to incorporate a extention method for implementing dependency injection in the project. This method should be explained in detail to ensure proper implementation.
![image](https://github.com/IT-Afra/bakery-shop/assets/99765069/725b1817-fbee-462f-9b7b-d050cbab0300)

**Comment by mohammadpirzadeh** [comment, marked as answer]
Hello
at the beginning
It is better to create a separate file for mappers and another file for services in the infrastructure section
Then we produce mappers and services.
After that, we create each one in each part of the program that we needed

**Reply by itafra-net** [reply]
This is the best answer for this problem. Thanks, Mohammad.
